solutions/product_solutions/NetInsight_Nimbra_Vision/01_netinsight-nimbra-vision-install.md

```diff
@@ -0,0 +1,68 @@
+---
+uid: netinsight-nimbra-vision-install
+---
+
+# Installing the DataMiner NetInsight Nimbra Vision solution
```

Comment:
This title calls it a solution, but in the table of contents it's called an app. We'll need to pick one or the other, because the title and the table of contents need to match.